EIPS/eip-3475.md

```diff
@@ -22,131 +21,127 @@ Current LP token is a simple ERC-20 token, which has not much complicity in data
 
 New functions built in ERC-3475 Multiple Callable Bonds Standard, will allow the users to economize their gas fee spend. Trading and burning of ERC-3475 Bonds will also multiply tokens market cap, helping it to recover from recession period. Existing structures, such as AMM exchanges or lending platform can be updated to recognize ERC-3475 Bonds.
 
+ERC-3475 bonds can also be batched calls for issuance and in turn be divided and exchanged in a secondary market.The defined function in this  standard do operations on the serialized array  reads/writes  on the array in the contract , rather than storing  in struct mappings ,giving significant benefits in storage and query gas costs.
```

Comment:
```suggestion
These bonds can also be batch called for issuance and, in turn, be divided and exchanged in a secondary market. The defined function in this standard do operations on the serialized array reads and writes on the array in the contract, rather than storing in struct mappings, providing significant benefits in storage and query gas costs.
```